.github/workflows/loristest.yml

```diff
@@ -71,6 +71,11 @@ jobs:
       if: steps.composer-cache.outputs.cache-hit != 'true'
       run: composer install --prefer-dist --no-progress --no-suggest
 
+    - name: Setup Node.js
```

Comment:
Is this required in main ? I don't think we have any issues with the tests.

Reply:
This section solely addresses the correction of the node version in the GitAction testing environment. It is a prerequisite for the main functionality, despite not having made any changes to the Loris core code.

Reply:
should I just delete that one row ?

Reply:
So node v18 is not supported if I understand well? Should it be specified somewhere in the install notes?

Reply:
someone should verify I guess?

Reply:
Yes, I did. See my edit above.

Reply:
Please delete them. The git action appears to be functioning properly.